src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset personal-info">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              autocomplete="disabled"
```

Comment:
http://htmlbook.ru/html/form/autocomplete

```suggestion
              autocomplete="off"
```